app/controllers/api/v1/targets_controller.rb

```diff
@@ -4,6 +4,11 @@ class TargetsController < ApplicationController
 
       before_action :authenticate_user!
 
+      def index
+        @targets = Target.where(user_id: current_user.id)
+        render json: { targets: @targets }, status: :ok
```

Comment:
As @santir489 mentioned in the previous PR, try creating a view for this controller action and define the target view at `app/views/targets/index.json.jbuilder`, instead of rendering here. Take a look at [this](https://github.com/rails/jbuilder).